lib/src/core/aspect.ts

```diff
@@ -1,11 +1,11 @@
 import {Pointcut} from './pointcut';
 
-export let AspectRegistry: { [name: string]: Aspect; } = {};
+export let AspectRegistry: Map<string, Aspect> = new Map<string, Aspect>();
```

Comment:
`export let AspectRegistry: Map<string, Aspect> = new Map<string, Aspect>();`
->
`export let AspectRegistry = new Map<string, Aspect>();`

Reply:
Went with consistency with the `Targets` set at line 5. Fixed that one as well.